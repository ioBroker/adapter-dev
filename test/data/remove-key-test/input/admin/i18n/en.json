{
    "hello": "Hello World",
    "goodbye": "Goodbye World",
    "removeMe": "This key will be completely removed"
}