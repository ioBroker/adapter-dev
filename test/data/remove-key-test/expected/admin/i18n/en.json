{
    "goodbye": "Goodbye World",
    "hello": "Hello World"
}